jormungandr/src/blockchain_stuck_notifier.rs

```diff
@@ -0,0 +1,50 @@
+use crate::{blockchain, utils::task::TokioServiceInfo};
+use chain_time::{
+    era::{EpochPosition, EpochSlotOffset},
+    Epoch,
+};
+use futures::prelude::*;
+use std::time::{Duration, SystemTime};
+use tokio::timer::Interval;
+
+pub fn check_last_block_time(
+    service_info: TokioServiceInfo,
+    blockchain_tip: blockchain::Tip,
+) -> impl Future<Item = (), Error = ()> {
+    let logger = service_info.logger().clone();
+
+    // TODO set from configuration
+    let check_period = Duration::from_secs(180);
```

Comment:
I am not sure about how the interval should be configured here?

Reply:
Make it a parameter on the node config file.